activerecord/lib/active_record/attributes.rb

```diff
@@ -132,21 +132,31 @@ module ClassMethods
       #     end
       #   end
       #
-      #   # config/initializers/types.rb
-      #   ActiveRecord::Type.register(:money, MoneyType)
-      #
       #   # app/models/store_listing.rb
       #   class StoreListing < ActiveRecord::Base
-      #     attribute :price_in_cents, :money
+      #     attribute :price_in_cents, MoneyType.new
       #   end
       #
       #   store_listing = StoreListing.new(price_in_cents: '$10.00')
       #   store_listing.price_in_cents # => 1000
       #
       # For more details on creating custom types, see the documentation for
-      # ActiveModel::Type::Value. For more details on registering your types
-      # to be referenced by a symbol, see ActiveRecord::Type.register. You can
-      # also pass a type object directly, in place of a symbol.
+      # ActiveRecord::Type::Value.
+      #
+      # It is also possible to register a type within an initializer. It is then possible to use a symbol to refer to
+      # the type rather than passing an instantiated type directly within attribute:
+      #
+      #   # config/initializers/types.rb
+      #   require 'lib/money_type' # NB: don't use an autoloaded location from initializers
+      #   ActiveRecord::Type.register(:money, MoneyType)
+      #
+      #   # app/models/store_listing.rb
+      #   class StoreListing < ActiveRecord::Base
+      #     attribute :price_in_cents, :money
+      #   end
+      #
+      # see ActiveRecord::Type.register for more details
+
```

Comment:
🎨 It's missing `#` here.

Reply:
Do you mean on L158? No I don't think so, it is the conclusion of the main doc comment and not part of the code example in lines 149-156.

Reply:
@feliperenan We could just drop line 158. It's kind of obvious that you need to see ActiveRecord::Type.register for more details

Reply:
We already have some titles in the documentation that help us with topics, so I would keep the `#` in all lines 😄.

I'm ok with removing the line 158 as well 👍.

Reply:
Sorry @timdiggins, I think I was not clear 😅 .

My comment was about line [159](https://github.com/timdiggins/rails/blob/248a2c15baf5beef651ac797ff46f3d12a9fb98c/activerecord/lib/active_record/attributes.rb#L159). Looking better for it now, we could just remove that line.

"I would keep # in all lines", means:

```
#
# see ActiveRecord::Type.register for more details
#
# ==== \Querying
```

Reply:
@feliperenan thanks for the clarification, now everything you said makes sense.